designer/client/src/common/ProcessUtils.ts

```diff
@@ -171,36 +179,51 @@ class ProcessUtils {
         }
     };
 
-    findNodeObjectTypeDefinition = (node: NodeType, processDefinition: ProcessDefinition): NodeObjectTypeDefinition => {
-        const foundDefinition = this.findDefinitionsForType(processDefinition, node)?.[this.findNodeDefinitionId(node)];
+    findComponentDefinition = (node: NodeType, components: Record<string, ComponentDefinition>): ComponentDefinition => {
+        const foundDefinition = components?.[this.findComponentId(node)];
         const emptyDefinition = {
             parameters: null,
             returnType: null,
         };
         return foundDefinition || emptyDefinition;
     };
 
-    private findDefinitionsForType = (processDefinition: ProcessDefinition, node?: NodeType): Record<string, NodeObjectTypeDefinition> => {
+    findComponentId = (node?: NodeType): string | null => {
```

Comment:
```suggestion
    resolveComponentId = (node?: NodeType): string | null => {
```
i'm not familiar with TS naming schemes, but to me `find` doesn't fit, it's all done on static information, no searching required

Reply:
Yeah, `find` is confusing in this context, but I think the `determine` word will be a little better than `resolve`, we use it quite often on the BE side in similiar context.